README.md

```diff
@@ -1,2 +1,55 @@
-# supervision-success-component
+# Supervision Success
+
 A React component powering a miniature web app for exploring the impact of success in supervision on the justice system at large
+
+## Contents
+
+1. [Input File Format](#input-file-format)
+
+## Input File Format
+
+Example of correct CSV could be found in [`params.csv`](/src/params.csv) file.
+
+First row of the table consists of next required params **in any order** (This params are used in application):
```

Comment:
Let's reword this line to, "The first row of file (the headers) contains the required revocation model parameters, **in any order**."